mne/epochs.py

```diff
@@ -394,6 +394,7 @@ def load_data(self):
         self._decim = 1
         self._raw_times = self.times
         assert self._data.shape[-1] == len(self.times)
+        self._raw = None  # shouldn't need it anymore
```

Comment:
Yes, `preload=True` eventually goes to `BaseEpochs.__init__` where it does:
```
        if preload_at_end:
            assert self._data is None
            assert self.preload is False
            self.load_data()  # this will do the projection
```